src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -75,9 +96,12 @@ class NavigationDrawer extends Component<CombinedNavigationProps> {
         id={`plugin-link-${plugin.link.replace(/\//g, '-')}`}
         button
       >
+        {imgSrc && (
+          <img className={this.props.classes.menuLogo} alt="" src={imgSrc} />
```

Comment:
Shouldn't `alt` have `logoAltText` set here?

Reply:
I didn't set `alt={logoAltText}` as the formatting of the image alt text (when it displays) and the rest of the text for the list item didn't match, instead I just added `logoAltText` to the start of `plugin.displayName` when a `src` for the image couldn't be found. Though thinking about it now I suppose this wouldn't cover the case where we do provide an image src, it just fails to load, or any accessiblity effects of having `alt` set. Do you think it would be better not to complicate things and just set `alt` with `logoAltText` and leave `plugin.displayName` as is?

Reply:
I was more thinking about the accessibility implications even if the logo is displayed. I kind of like the use of logoAltText to be prepended to the display name if there's no logo though.